connectors/embeddings-vector-database/pom.xml

```diff
@@ -25,6 +25,7 @@
 
   <properties>
     <version.apache-tika>3.2.3</version.apache-tika>
+    <elasticsearch-rest-client.version>9.2.2</elasticsearch-rest-client.version>
```

Comment:
not managed by spring anymore see https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-4.0-Migration-Guide#elasticsearch-client